static/js/settings_notifications.js

```diff
@@ -203,7 +211,34 @@ export function set_up(settings_panel) {
     });
 
     set_enable_marketing_emails_visibility();
+
     rerender_ui();
+
+    // Has to be after rerender_ui because relevant rows in table are added there
+    container.find(".unmute_stream").on("click", (e) => {
+        const row = $(e.currentTarget).closest(".stream-row");
+        const stream_id = Number.parseInt(row.attr("data-stream-id"), 10);
+        const sub = sub_store.get(stream_id);
+
+        stream_settings_ui.set_muted(sub, !sub.is_muted);
+
+        $(row).toggleClass("control-label-disabled", false);
+        $(row).find(".unmute_stream").hide();
+        $(row).find("input[disabled]").prop("disabled", false);
+
+        display_saving_msg(row);
```

Comment:
We actually do not need this. `set_muted` function accpets status element, you can just pass the status element to it and it will take care of showing the `saving` message.